### DISCUSSION TITLE
high-scores/players/kingmidas666/

### DISCUSSION THREAD

**Discussion by giscus** [stb-gaming/high-scores#131] [Comments]
# high-scores/players/kingmidas666/

Sky Games High score

https://stb-gaming.github.io/high-scores/players/kingmidas666/

**Comment by tjkayes6** [comment]
Hey will you add missing Sky Games like 21's
24: Daily Quiz
Aladdin 1
Aladdin: Gone to Pieces
Alice in Wonderland
All Grown Up: Confiscated Cards
American Dragon Jake Long: Big Apple Rescue
Are You Smarter than a 10 Year Old?
Asteroids
Avatar the Last Airbender: The Legend of Aang
Avatar 2
Banzai Pinball
Battleship
Beehive Bedlam (original)
Beehive Bedlam (Denki) (unreleased port)
Beehive Bedlam 2
Ben 10 Episode 1: Vilgax Venom
Ben 10 Episode 2: Call of the Wildmutt
Ben 10 Episode 3: Upchuck Unveiled
Ben 10 Episode 4: Howlin' Benwolf
Ben 10 Alien Force Episode 1: Swampfired Up!
Ben 10 Alien Force Episode 2: Operation D'Void
Ben 10 Alien Force Episode 3: Humungosaur Smash!
Big Brother 3
Big Brother: Chicken Feed
Big Top Drop
Bips!
Blockbusters
Boggle
Boys & Girls
Bumble Tumble
Bust-a-Move
Carol Vorderman's Mind Aerobics
Carol Vorderman's Mind Aerobics: Summer Shape Up
Cars
Cartoon Carnage
Caterpillar Crunch
Celebrity Shagmatch
Centipede
Chess Champion
Chicken Little
Chowder: The Upset Oozleberry
Chowder 2
Chroma
The Chronicles of Narnia: The Lion, the Witch and the Wardrobe
Cluedo
Codename: Kids Next Door Episode 1: Operation: B.I.T.T.E.R.S.W.E.E.T.
Codename: Kids Next Door Episode 2: Operation: S.H.I.P.W.R.E.C.K.E.D.
Codename: Kids Next Door Episode 3: Operation: R.E.V.E.N.G.E.
Codename: Kids Next Door Episode 4: Operation: G.U.M.D.R.O.P.
Codename: Kids Next Door: Operation: T.E.E.N.A.G.E.R.
Connect 4
Coronation Street
Corporal Cluck
Courage the Cowardly Dog Episode 1: Katz Kidnap
Courage the Cowardly Dog Episode 2: Katz Komeback
Courage the Cowardly Dog Episode 3: Le Quack Attack
Cow and Chicken
Crazy Golf
Crazy Golf: Putting on Pluto
Crazy Sundaes
Crunch Time
Darts
Deal or No Deal
Deal or No Deal: Gold
Denki Blocks
Denki's Sticky Blocks!
Dexter's Laboratory Episode 1: Dexter vs. Mandark
Dexter's Laboratory Episode 2: Mandark Strikes Back
Dexter's Laboratory Episode 3: Dorkster's Revenge
Dexter's Laboratory: Plasma Panic!
Dexter's Laboratory: Robot Rampage
Dice
Don't Forget the Lyrics
Double Up
Duopolis
Ed, Edd n Eddy: Ed's Crazy Golf
Ed, Edd n Eddy: Jawbreaker Pinball
Ed Edd n Eddy: Jawbreakers Shakedown
Ed, Edd n Eddy Episode 1: Night of the Living Ed
Ed, Edd n Eddy Episode 2: Jingle-Bell Ed
Elevens
Elevenses
Family Fortunes
Family Fortunes: Celebrities
Family Fortunes: Gold
Fanboy & Chum Chum: The Phangum Menace
Fathom
Fathom II
Finding Nemo
Finding Nemo 2
Finding Nemo III
Finding Nemo
Flaunt: Daily Quiz
Flip It
Fosters Home for Imaginary Friends Episode 1: Tick Tock Terror
Fosters Home for Imaginary Friends Episode 2: Brotherly Love
Fox & Pounds
Frontline
Gem Quest
Generator Rex
Genie's Cave
Gin Runny
Gladiators
Gladiators: Series 2
Gold Runner
Golden Balls
Graham Norton Volleyball
The Grim Adventures of Billy & Mandy
The Grim Adventures of Billy & Mandy: Hell's Hamsters
Grim's Cookie Quest
Hangman: Celebrity Gunk Tank
Hannah Montana: Secret Songs
Herbie Fully Loaded
High School Musical
High School Musical: Encore Edition
High School Musical: Act 3
Hive Jive
Hot Wheels Monster Truck Mayhem
Hulk
I'm a Celebrity: Get Me Out of Here!
iCarly: iHate Pesky Robots
The Incredible Hulk (unreleased port)
The Incredibles
The Incredibles: Bomb Voyage
The Incredibles III: Rise of the Underminer
Intergalactic: Daily Quiz
Johnny Bravo in Pop's Burger Panic
Jumble
Jumble Fever
Plus
Avatar
Beehive
Cartoon Network (unreleased)
Christmas
Easter
Football
Halloween
Scooby-Doo
Shrek
SpongeBob SquarePants
Tom and Jerry
Jumble Quest: The Lost Pyramids
The Jungle Book
The Jungle Book 2
Jump In
Kim Possible: Gone Ron
Kim Possible: Robot Frenzy
King Tutti
Kingpin Bowling
LEGO Castle: The Sword of Virtue
Letter Box
Lilo & Stitch: Island Racers
Lilo & Stitch: Obhana Cup
Looney Tunes: Back in Action Episode 1
Looney Tunes: Back in Action Episode 2
Looney Tunes: Back in Action Episode 3: Rabbit Rescue
Madagascar: Plane Crazy
Mah Jong
Meet the Robinsons: Mind Mechanic
Mickey's Lost World Adventure
Monopoly
Monopoly Deluxe
Monopoly Quiz
Monsters, Inc.
The Muppet Show: Backstage Rampage
The Nightmare Before Christmas
Number Crunch
Ocean Gin
Offroad Racing
Pac-Man
Perfect Match (Open....)
Perfect Match (Gamestar)
Perfect Match: Masterclass
Peter Pan: Lost Boy Rescue
Pirates of the Caribbean: At World's End
Pirates of the Caribbean: Dead Man's Chest
Pirates of the Caribbean: The Curse of the Black Pearl
Play Your Cards Right
Poker
Pong
Pool
Poolside Palaver
Pop 'n' Drop
Pop 'n' Drop Galaxy
The Powerpuff Girls: Crystal Crisis
The Powerpuff Girls: Mojo for Mayor
The Powerpuff Girls: Townsville Attack
The Princess and the Frog
Pub Quiz
Puzzler: Crosswords
Puzzler: Sudoku
Puzzler: Wordsearch
Pyramid Solitaire
Quids Inn
Recess: Tunnel Rescue
Recess: Tunnel Rescue 2 Christmas
Recess: Tunnel Rescue 3
Recess 2: Disarm Alarm
Robotboy Episode 1: Kamikazi Kidnap!
Robotboy Episode 2: Something in the Darkness Drools!
Robotboy Episode 3: Repair or Despair
Robotboy Episode 4: Tag Team Trouble
Scalextric
Scooby Doo: Freaky Fairground
Scooby-Doo: Ghoul Duel
Scooby-Doo! Episode 1: The Mystery of Eerie Isle
Scooby-Doo! Episode 2: Mummy Madness
Scooby-Doo! Episode 3: The Curse of Mystery Mine
Scooby-Doo! Episode 4: Tomb It May Concern
Scooby-Doo! Mysteries Episode 1: The Case of the Creepy Cowboy
Scooby-Doo! Mysteries Episode 2: The Case of the Frosty Snowman
Scooby Dooby Boom!
Scrabble
Scuzz Pinball
Scuzzboarding
Sheep Dip
Shrek Episode 1: Fiona's Rescue
Shrek Episode 2: Disarming Charming
Shrek Episode 3: Fairy Godmother's Revenge
Shrek Episode 4: Double Trouble
Shrek Episode 5: Imperial Peril
Shrek Jumble Rumble
Shrek Forever After
Shrek the Third
The Simpsons
Sky Movies Quiz
Sky Sports Batting Challenge
Sky Sports Cricket Ashes
Sky Sports Daily Football Quiz
Sky Sports Premiership Football Challenge
Sky Sports Tee Time Golf
Soccer AM Car Park Challenge
Solitaire
Solitaire Cruise
Space Invaders
Speed Draughts
SpongeBob SquarePants Episode 1: Bikini Bottom Bother
SpongeBob SquarePants Episode 2: Krabby Patty Kidnap
SpongeBob SquarePants Episode 3: The Flying Dutchman
SpongeBob SquarePants Episode 4: Save Our Sandy
SpongeBob SquarePants Episode 5: Snow Woes
SpongeBob SquarePants Episode 6: Forest of Fear
SpongeBob Jellyfishing
Star Trek The Next Generation: STRANDED
String 'em In
String 'em In 2
The Suite Life of Zack and Cody
Super Breakout
Takeshi's Castle
That's So Raven
Thunderbirds: Flashpoint Earth
Timon & Pumbaa: Bug Feast
Tomb Raider Apocalypse: Episode 1: The Eye of Osiris
Tomb Raider Apocalypse: Episode 2: The Shadow Falls
Tomb Raider Apocalypse: Episode 3: The Last Midnight
Tomb Raider: The Temple of Anubis
Tom and Jerry Episode 1: Mouse Party
Tom and Jerry Episode 2: Merry Christmouse
Tom and Jerry Episode 3: Mansion House Mouse
Tom and Jerry Episode 4: Duck and Cover
Tom and Jerry Episode 5: Rickety House Mouse
Tom and Jerry Episode 6: An Eggs-plosive Easter
Tom and Jerry in Dynamice
Tom and Jerry in Dynamice 2
Tom and Jerry in Food Fight
Tom and Jerry in Food Fight 2
Tom and Jerry in Food Fight 3
Tom and Jerry in Mouse in the House
Top Trumps: Dragon Booster
Top Trumps: Supercars
Toy Story
Trivial Pursuit
V Birds
Wall-E
Wallace & Gromit: The Curse of the Were-Rabbit
Who Wants to be a Millionaire
70's
Athens
Celebrity
Christmas
Comedy
Euro 2004
Football
Gold
Headlines
Junior
Movies
Music
Showbiz
Soaps
Sports
TV
The Wild
Winnie the Pooh: Yunny Hunny
Word Crunch
Word Crunch 2: Conundrums
Word Crunch 3: Conundrums (Gold)
Word Crunch Deluxe
Word Punch
Word Search
Worms
Check the games that are already added and don't forget to add more games to STB Github's Sky Games menu.
